### PR TITLE
draft of criteria for using your own data

### PR DESCRIPTION
**Pull request by srappel**
Leaving this as a pull request in case anyone has suggestions for criteria for if people wnat to use their own data.

### CODE REVIEW

**Comment by srappel** [comment]
Change "Provided Data" to "Capstone Data"
"Your Data"
"Lesson data"

**Review by mkamenet3 — COMMENTED** [review]
I think these changes look great, I really like the tips for success! No additional changes suggested.

File: setup.md [hunk 1]
```diff
@@ -56,11 +51,19 @@ You can read more about the BIOCLIM dataset
...
 ## Using your own data
 
-If you chose your own data for this workshop, the data should meet these criteria:
+Advanced participants can replace some or all of the capstone data with their own data. Below are some tips for success:
+
+* Be familiar with the data, its formatting, and what variables and values mean.
+
+* All data should have the same or similar geographic extents. If you plan to use a mix capstone data with your data, your data should cover Wisconsin.
+
+* Data should be georeferenced and projected. GeoTIFF format is recommended.
```
Comment (sjgraves) [inline setup.md]: Can specify GeoTIFF for raster and shapefiles for vectors

**Review by sjgraves — COMMENTED** [review]
I like the details of using your own data! I made one comments about specifying the file format. Otherwise it looks good.

**Pull request merged**